examples/source/interactivity-dynamic-content/Subscription_Settings/index.html

```diff
@@ -0,0 +1,434 @@
+
+      We ensure the `select` element's selected `option` stays in sync with `currentSubscription` by binding the `selected` attribute of each `option`. Finally, we revert the `select` element to its previous value on form submission failure by setting `currentSubscription` to `previousSubscription` in the `form` element's `submit-error` event.
+
+      The following code snippet is configured to fail form submissions. Test out the new behavior!
+     -->
+    <amp-list src="<% base_path %>/subscription" binding="refresh" items="." single-item layout="fill">
+      <template type="amp-mustache">
+        <form action-xhr="<% base_path %>/subscription?fail=true" method="post" id="form3"
+          on="submit-success: AMP.setState({ nextSubscription: null });
+            submit-error:
+              AMP.setState({
+                currentSubscription: previousSubscription,
+                nextSubscription: null
+              });">
+          <input type="hidden" name="nextSubscription" [value]="nextSubscription">
+          <select
+            on="change:
+                AMP.setState({
+                  previousSubscription: currentSubscription || '{{currentSubscription}}',
+                  currentSubscription: event.value,
+                  nextSubscription: event.value
+                }),
+                form3.submit;"
+            [disabled]="!!nextSubscription">
+            {{#options}}
+              {{#isSelected}}
+                <option value="{{value}}" [disabled]="!!nextSubscription"
+                  selected [selected]="(currentSubscription || '{{currentSubscription}}') == '{{value}}'">
+                  {{text}}
+                </option>
+              {{/isSelected}}
+              {{^isSelected}}
+                <option value="{{value}}" [disabled]="!!nextSubscription"
+                  [selected]="(currentSubscription || '{{currentSubscription}}') == '{{value}}'">
+                  {{text}}
+                </option>
+              {{/isSelected}}
+            {{/options}}
+          </select>
+          <div submit-error>Something went wrong. Please try again</div>
+        </form>
+      </template>
+      <div placeholder>Loading...</div>
+      <div fallback>Something went wrong. Please refresh</div>
+    </amp-list>
+
+    <!--
+      #### Step 6: Icon Dropdown
+
+      Styling the `select` element itself to look like an icon using only CSS supported by email clients isn't feasible. Instead, we'll hide the `select` element's persistent user interface, which shows the element's current value, and show a different icon for each dropdown state in its place, but keep the element's ephemeral dropdown, what you see on click.
```

Comment:
The second sentence in this section is a little long and confusing. Could you try breaking it up for clarity?

Reply:
Done!